docs/api.md

```diff
@@ -2901,14 +2909,13 @@ page.on('requestfailed', request => {
 - returns: <[Promise]>
 
 Fulfills request with given response. To use this, request interception should
-be enabled with `page.setRequestInterception`. Exception is thrown if
+be enabled with `page.route`. Exception is thrown if
```

Comment:
This is cumbersome if I want to keep my existing event-based code working with minimal changes. Do I have to replace `page.setRequestInterception` with `page.route('*', () =>{})` ? Maybe we can support `page.route()` call for that (also imperfect?

Reply:
There is no real migration yet, we are at 0.10, but in terms of Puppeteer parity, you delete the `page.setRequeatInterception` call and replace `page.on` with `page.route` instead. Empty `page.route()` does not make sense - it stalls the world.

I don't expect `request.continue` to be called from event ever...